bin/mail/2016-03-21-teacher-recruitment/generate-inactive-teachers

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env ruby
+require_relative '../mailing-common/mailing-list-utils'
+require 'set'
+
+# Inactive Code Studio teacher accounts (not teaching a course, or haven't added students)	US only
+
+US_TEACHERS = Set.new
+SOLR_PARAMS = {
+  q: 'kind_s:user && role_s:teacher',
+  fq: 'create_ip_country_s:"United States"',
+  fl: 'email_s',
+  rows: 10000
```

Comment:
Thanks! Obviously, I still have a lot to learn (as well as a distrust of copying existing patterns).